Dockerfile

```diff
@@ -3,24 +3,24 @@
 # Copyright Authors of Cilium
 # SPDX-License-Identifier: Apache-2.0
 
-# FINAL_CONTAINER specifies the source for the output
-# cilium-cli-ci (default) is based on ubuntu with cloud CLIs
-# cilium-cli is from scratch only including cilium binaries
-ARG FINAL_CONTAINER="cilium-cli-ci"
-
 FROM docker.io/library/golang:1.23.2-alpine3.19@sha256:f6392ffebb028fed5ffe743ddb9716e38402c978779edd66474bb5d05f5e65e4 AS builder
 WORKDIR /go/src/github.com/cilium/cilium-cli
 RUN apk add --no-cache curl git make ca-certificates
 COPY . .
 RUN make
 
+# cilium-cli is from scratch only including cilium binaries
 FROM scratch AS cilium-cli
-COPY --from=builder /go/src/github.com/cilium/cilium-cli/cilium /usr/local/bin/cilium
 ENTRYPOINT ["cilium"]
+LABEL maintainer="maintainer@cilium.io"
```

Comment:
Doing the label before copying the binaries improves cacheability of the layer as it will only require the last 2 layers to be replaced on the final image.

Reply:
nit: This seems like a noop to me. I would be highly surprised if the label/entrypoint commands took any significant amount of resources to execute. But :shrug: I don't see harm in shuffling these around. It's a little more tidy.